truffle/src/main/java/org/jruby/truffle/RubyFiletypeDetector.java

```diff
@@ -0,0 +1,50 @@
+package org.jruby.truffle;
```

Comment:
I’ve forgot to ask, `RubyFiletypeDetector` is currently in `org.jruby.truffle` package, not the best place. What about `org.jruby.truffle.runtime.util` or other new `org.jruby.truffle.util`?